src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/util/RandomData.kt

```diff
@@ -21,3 +24,9 @@ fun randomPostCode() = randomStringUpperCase(2) + randomNumberChars(1) + " " +
   randomStringUpperCase(2) + randomNumberChars(1)
 
 fun randomInt(min: Int, max: Int) = Random.nextInt(min, max)
+
+fun randomDateAfter(point: LocalDate): LocalDate = point.plusDays(randomInt(1, 14).toLong())
+fun randomDateBefore(point: LocalDate): LocalDate = point.minusDays(randomInt(1, 14).toLong())
+
+fun randomDateTimeAfter(point: OffsetDateTime): OffsetDateTime = point.plusMinutes(randomInt(1, 60 * 24 * 14).toLong()).truncatedTo(ChronoUnit.SECONDS)
+fun randomDateTimeBefore(point: OffsetDateTime): OffsetDateTime = point.minusMinutes(randomInt(1, 60 * 24 * 14).toLong()).truncatedTo(ChronoUnit.SECONDS)
```

Comment:
These feel like they would make nice extension functions.

Reply:
Good shout, have switched them to extension functions